macros/parsers/parserLinearInequality.pl

```diff
@@ -104,16 +104,7 @@ sub Init {
 			string        => ' <= ',
 			TeX           => '\le ',
 			kind          => "le",
-			class         => 'LinearInequality::inequality',
-			formulaClass  => "LinearInequality"
-		},
-		'=<' => {
-			precedence    => .5,
-			associativity => 'left',
-			type          => 'bin',
-			string        => ' <= ',
-			TeX           => '\le ',
-			kind          => "le",
+			alternatives  => [ '=<', "\x{2264}" ],
```

Comment:
I'm confused by the alternatives, as the first one is the same as the actual operator at line 10.  Was this meant to go in the previous operator `<=`?

One could make `=<` an alias for `=>` rather than an alternative.  I think that might not have been available for operators when this was written.

Reply:
Wait, I'm looking at this again and I think it is right. Maybe the diff is misleading.

This removes `=<` as an operator altogether. And then it makes `=<` and `"\x{2264}"` be alternatives for `<=`. Because the original block for `<=` and `=<` are right next to each other and nearly identical, the diff that is displayed might be misleading.

Reply:
OK, I didn't think about how alternatives are not available by default. I was thinking I was just simplifying the code, but of course you are right that it would change behavior. I updated the commit to only add the ≤ and ≥ alternatives. (And ≠ in the other file where `!=` has meaning.)